internal/ignition/ignition.go

```diff
@@ -1053,12 +1053,12 @@ func writeIgnitionFile(path string, config *config_latest_types.Config) error {
 	return nil
 }
 
-func setFileInIgnition(config *config_latest_types.Config, filePath string, fileContents string, appendContent bool, mode int) {
+func setFileInIgnition(config *config_latest_types.Config, filePath string, fileContents string, appendContent bool, mode int, overwrite bool) {
```

Comment:
It's better to use a type for bool in this case, something like https://github.com/openshift/assisted-service/blob/af0bafb3f7f629932f8c3dc31ccddedfe6984926/internal/common/db.go#L126-L131 
It makes the code more readable when you have a long list of parameters